tests/test_utils.sh

```diff
@@ -694,6 +694,72 @@ function test_path_format_last_dir() {
   assertEquals "full directory formatting with multichar separator" "{l}pathname" "$lp_path_format"
 }
 
+function test_lp_fill {
+    local lp_fill
+    echo $COLUMNS
+    _lp_fill "Left" "Right" " "
+    assertEquals "full width" ${COLUMNS} ${#lp_fill}
```

Comment:
This test is failing in the runners because `$COLUMNS` is not set in environments without a terminal.

Just set `$COLUMNS` to something sane, like 60.